src/styles/_theme.scss

```diff
@@ -8,8 +8,8 @@
    feel free to play around with these and see what other look you can come up with! :D
 */
 
-$accent: #07b699;
-$accentDark: #068383;
+$accent: #cde9e1e8;
+$accentDark: #bbecdedc;
 $textColor: white;
```

Comment:
I would adjust the text color to something darker. It's quite difficult to read...